Tools/Pipeline/Windows/MainView.cs

```diff
@@ -77,6 +78,9 @@ public void Attach(IController controller)
             _controller.OnProjectLoading += invokeUpdateMenus;
             _controller.OnProjectLoaded += invokeUpdateMenus;
 
+            _controller.OnProjectLoading += OnProjectLoading;
+            _controller.OnProjectLoaded += OnProjectLoaded;
```

Comment:
Since they are really never removed.  You could do this as an inline delegate and make the intent more clear.